packages/web-client/app/components/card-space/profile-card/index.css

```diff
@@ -0,0 +1,108 @@
+.card-space-profile-card__container {
+  --profile-card-cover-photo-width: 17.5rem;
+  --profile-card-cover-photo-height: calc(var(--profile-card-cover-photo-width) * 50 / 125);
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card-space-profile-card__preview {
+  margin-bottom: var(--boxel-sp-xxs);
+  color: var(--boxel-purple-400);
+  font: 600 var(--boxel-font-xs);
+  letter-spacing: var(--boxel-lsp-xl);
+  text-transform: uppercase;
+}
+
+.card-space-profile-card__card {
+  position: relative;
+  width: var(--profile-card-cover-photo-width);
+  border-radius: var(--boxel-border-radius);
+  overflow: hidden;
+  box-shadow: 0 15px 30px rgba(0, 0, 0, 0.2);
+}
+
+.card-space-profile-card__cover-photo-container {
+  margin: 0;
+}
+
+.card-space-profile-card__cover-photo {
+  height: var(--profile-card-cover-photo-height);
+  width: 100%;
+  object-fit: cover;
+}
+
+.card-space-profile-card__cover-photo-container--placeholder {
+  background-color: var(--boxel-light-400);
+}
+
+.card-space-profile-card__cover-photo--placeholder {
+  --icon-color: var(--boxel-purple-300);
+
+  transform: scale(0.7);
+}
+
+.card-space-profile-card__cover-photo--placeholder > circle {
+  display: none;
+}
+
+.card-space-profile-card__profile-photo-container {
+  --image-height: 3rem;
+
+  position: absolute;
+  top: calc(var(--profile-card-cover-photo-height) - var(--image-height) / 2);
+  height: var(--image-height);
+  width: var(--image-height);
+  border-radius: var(--image-height);
+  border: 1px solid var(--boxel-purple-300);
+  left: var(--boxel-sp);
+  background: var(--boxel-light);
+  margin: 0;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card-space-profile-card__profile-photo-container--placeholder {
+  background: var(--boxel-light-400);
+}
+
+.card-space-profile-card__profile-photo--placeholder {
+  --icon-color: var(--boxel-purple-300);
+
+  transform: scale(0.5);
+}
+
+.card-space-profile-card__body {
+  padding: var(--boxel-sp-lg) var(--boxel-sp) var(--boxel-sp);
+}
+
+.card-space-profile-card__heading {
+  margin-top: var(--boxel-sp);
+  font: 700 var(--boxel-font-lg);
+}
+
+.card-space-profile-card__host {
+  font: 600 var(--boxel-font-sm);
+  color: var(--boxel-purple-500);
+}
+
+.card-space-profile-card__category {
+  display: inline-block;
+  margin-top: var(--boxel-sp-xxxs);
+  padding: 2px var(--boxel-sp-xxxs) 2px var(--boxel-sp-xxxs);
+  color: var(--boxel-purple-300);
+  font: 600 0.55rem var(--boxel-font-family);
```

Comment:
Is there a possibility of having `--boxel-font-xxs`? I looked at adding it but I couldn’t understand the ratios between sizes (and line heights).

Reply:
I don't see why not, though I have doubts about whether we should use fonts that small.

Reply:
ya I would generally avoid it but it’s in the design… are you the person who set up the font variables and ratios, @burieberry? What should I do 😯